synapse/lib/cell.py

```diff
@@ -2228,6 +2228,22 @@ async def popUserProfInfo(self, iden, name, default=None):
         user = await self.auth.reqUser(iden)
         return await user.profile.pop(name, default=default)
 
+    async def getUserVars(self, iden):
+        user = await self.auth.reqUser(iden)
+        return user.vars.pack()
```

Comment:
Do we maybe want to make this a tuple yielder to account for eventually not having them in the hive and potential user var growth?

Reply:
updated to `iterUserVars`